model_path: "/home/quanvu/git/segmentation-module/deeplabv3_apples/output/2024_10_11_18_57_32/best_model.pt"
input_size:
  width: 800
  height: 800